Source/Events/AggregateRoot.cs

```diff
@@ -131,10 +131,10 @@ public virtual void ReApply(CommittedAggregateEvents events)
         /// <summary>
         /// Fast forward to the specified version of the <see cref="EventSource">EventSource</see>.
         /// </summary>
-        /// <param name="version">Version to fast foward to.</param>
+        /// <param name="version">Version to fast forward to.</param>
         public void FastForward(AggregateRootVersion version)
         {
-            ThrowIfStateful();
+            ThrowIfStatefull();
```

Comment:
It was correct before :)  Stateful not Statefull

```suggestion
            ThrowIfStateful();
```